docs/content/src/learn/tutorials/messaging-integrations/messaging-with-kafka/src/product_management_system/consumer1.bal

```diff
@@ -0,0 +1,35 @@
+import ballerina/kafka;
+import ballerina/io;
+import ballerina/lang.'string as strings;
+
+// CODE-SEGMENT-BEGIN: kafka_consumer_config
+kafka:ConsumerConfig consumer1 = {
+    bootstrapServers: "localhost:9092",
+    groupId: "consumer",
+    topics: ["product-price"],
+    pollingIntervalInMillis: 1000,
+    partitionAssignmentStrategy: "org.apache.kafka.clients.consumer.RoundRobinAssignor"
+};
+
+listener kafka:Consumer productConsumer1 = new (consumer1);
```

Comment:
Rename `productConsumer1` to a meaningful name.

Reply:
Done